Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "osm_boundaries_utils"
-version = "0.10.0"
+version = "0.11.0"
 authors = ["dt.ro <dt.ro@canaltp.fr>", "Antoine Desbordes <antoine.desbordes@gmail.com>"]
 description = "utilities to help reading OpenStreetMap boundaries in rust."
 repository = "https://github.com/Qwant/osm_boundaries_utils_rs"
```

Comment:
Can you change repo url ? And check other Qwant reference ?

Reply:
Nice catch. Changed in [e2472d8](https://github.com/hove-io/osm_boundaries_utils/pull/1/commits/e2472d8c252625e813efe5af21f01d4b778dc1dd) and no other reference found.